src/documentation/0028-npx/index.md

```diff
@@ -46,7 +46,7 @@ A typical demonstration of using `npx` is through the `cowsay` command. `cowsay`
 
 This only works if you have the `cowsay` command globally installed from npm previously. Otherwise you'll get an error when you try to run the command.
 
-`npx` allows you to run that npm command without having it installed locally:
+`npx` allows you to run that npm command without having it installed globally:
```

Comment:
I think npx first checks the bin in the local project, and then in the global $PATH

Reply:
Oh I understood that, but in this context (if you read the previous few lines), the author is trying to emphasize the fact that you can run a program from the terminal even if it isn't installed globally (found in the list of $PATH) by using npx.